ubl/Handlers/Common/PaymentMeansHandler.php

```diff
@@ -13,12 +13,107 @@ class PaymentMeansHandler extends UblHandler {
 	public function handle( $data, $options = array() ) {
 		$payment_means = array(
 			'name'  => 'cac:PaymentMeans',
-			'value' => array(),
+			'value' => $this->get_payment_means(),
 		);
 
-		$data[] = apply_filters( 'wpo_wc_ubl_handle_PaymentMeans', $payment_means, $data, $options, $this );
-
+		$data[] = apply_filters( 'wpo_wc_ubl_handle_PaymentMeans', $payment_means, $options, $this );
+		
 		return $data;
 	}
 
+	private function get_payment_means_code( $payment_method ) {
+		// Map WooCommerce payment methods to Payment Means Code
+		// All available codes: https://docs.peppol.eu/poacc/billing/3.0/2024-Q2/codelist/UNCL4461/
+		$mapping = array(
+			'cod'    => '10',
+			'bacs'   => '31',
+			'cheque' => '20',
+			'paypal' => 'ZZZ',
+			'stripe' => 'ZZZ',
```

Comment:
What do you think about `48`?

Reply:
It requires additional card details, which can be challenging to obtain.